static/sass/ds/7-utilities/_bg.scss

```diff
@@ -0,0 +1,19 @@
+.has-white-bg {
+  background-color: $color-white-pure;
+}
+
+.has-white-bg-from-bp-l {
+  @include mq($from: bp-l) {
+    background-color: $color-white-pure;
+  }
+}
+
+.has-white-off-bg {
+  background-color: $color-white-off;
+}
+
+.has-white-off-bg-until-bp-l {
+  @include mq($until: bp-l) {
+    background-color: $color-white-off;
+  }
+}
```

Comment:
The media query change totally make sense to me because we don't have classes for those. I'm confused about the need for renaming the `has-white-bg` and `has-white-off-bg` though. Want to chat through that?